test/util.ts

```diff
@@ -0,0 +1,179 @@
+import { readArchiveHeaderSync } from '@electron/asar/lib/disk';
```

Comment:
We shouldn't reach into `@electron/asar` internals here. Looks like this is already exposed [via the public API as `getRawHeader`](https://github.com/electron/asar/blob/b2f73c8e5ee9ef64106513c55e67f6fd5d1f576f/src/asar.ts#L240-L242), and that should handle the typings for us as well.

Reply:
Thank you for the feedback! Updated the PR to use the exposed public method